lib/pi_piper/sysfs/version.rb

```diff
@@ -0,0 +1,6 @@
+module PiPiper
+  class Driver; end
```

Comment:
Here's a thought for this `Driver` class. I think it would be better to put it either in `PiPiper` or in its own gem.
This is in case we want the Driver class to have any base methods that all drivers need. Plus it would be declared in one spot. We do `gem 'pi_piper'` in these gems anyway so maybe just stick the `Driver` class in  `PiPiper` for simplicity?